config/environment.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
-module.exports = function(/* environment, appConfig */) {
-  return { };
+/**
+ * Export the default config for ember-devtools. By default, enable only
+ * in development.
+ */
+module.exports = function(environment, appConfig) {
+  appConfig['ember-devtools'] = appConfig['ember-devtools'] || {};
+  appConfig['ember-devtools'].enabled = appConfig['ember-devtools'].enabled || environment === 'development';
+  appConfig['ember-devtools'].global = appConfig['ember-devtools'].global || false;
```

Comment:
I completely agree, however this is in a node environment and I don't believe we can use `Ember` here. The alternative would be to include something like [`lodash/object/merge`](https://github.com/ember-cli/ember-cli/blob/7af15c5f5e3ddbf8cdbde84d5b968a053c8c322d/lib/models/project.js#L14) or vendor https://github.com/emberjs/ember.js/blob/v1.12.0/packages/ember-metal/lib/merge.js.